pycm/pycm_compare.py

```diff
@@ -252,5 +252,7 @@ def __compare_assign_handler__(compare, cm_dict, class_weight, digit):
             if all([isfloat(x) for x in class_weight.values()]
                    ) and sum(class_weight.values()) != 0:
                 compare.class_weight = class_weight
+            else:
+                warn(COMPARE_CLASS_WEIGHT_WARNING,RuntimeWarning)
```

Comment:
A space after comma is suitable here I think.
`warn(COMPARE_CLASS_WEIGHT_WARNING, RuntimeWarning)`

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/386/commits/b0c7414b2f9f7a23e40d1de3be61797f0aa415c7